docs/source/user_guide/interacting_with_nodes.rst

```diff
@@ -186,7 +186,7 @@ In general, changes to a receptor configuration file do not take effect until th
 
 However, the action items pertaining to receptor backend connections can be reloaded, without a receptor restart. These include the following,
```

Comment:
I know this isn't part of the PR, but some clarification might be good here. Does this mean, "any action item related to Receptor backend connections can be reloaded, here's the list" or "some action items related to Receptor backend connections can be reloaded, others cannot, here's the list of ones that can".

Reply:
Great idea.  I clarified the text.  Additional suggestions welcome!